tsconst/interface.go

```diff
@@ -9,3 +9,4 @@ package tsconst
 // WintunInterfaceDesc is the description attached to Tailscale
 // interfaces on Windows. This is set by our modified WinTun driver.
```

Comment:
While we're in here:
"This is set by our modified WinTun driver."

Recommend: "This is set by the WinTun driver."

We de-forked wintun some time ago.